absl/base/CMakeLists.txt

```diff
@@ -243,7 +243,7 @@ absl_cc_library(
     ${ABSL_DEFAULT_COPTS}
 )
 
-absl_cc_library(
+absl_cc_test(
```

Comment:
Most of these changes (absl_cc_test) aren't correct. These are test libraries, not tests. Perhaps we need a way of differentiating test libraries.